extensions/http-curl/client/HTTPClient.h

```diff
@@ -222,6 +222,21 @@ class HTTPClient : public BaseHTTPClient, public core::Connectable {
       }
     }
   }
+ private:
+  static int onProgress(void *client, curl_off_t dltotal, curl_off_t dlnow, curl_off_t ultotal, curl_off_t ulnow);
+
+  struct Progress{
+    std::chrono::high_resolution_clock::time_point last_transferred_;
+    curl_off_t uploaded_data_;
+    curl_off_t downloaded_data_;
+    void reset(){
+      last_transferred_ = std::chrono::high_resolution_clock::now();
```

Comment:
Is high res clock steady clock?
Otherwise changes in system time or DST changes can make fun of us.